### ISSUE TITLE
Build fail on pinebook - armbian

### ISSUE THREAD

**Issue by luruke** [patriciogonzalezvivo/glslViewer#107]
Hello! tried to build on pinebook + armbian:

```
luruke@pinebook:~/git/glslViewer$ uname -a
Linux pinebook 3.10.107-pine64 #3 SMP PREEMPT Mon Jun 4 16:56:16 UTC 2018 aarch64 aarch64 aarch64 GNU/Linux
luruke@pinebook:~/git/glslViewer$ cat /etc/os-release
NAME="Ubuntu"
VERSION="16.04.5 LTS (Xenial Xerus)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 16.04.5 LTS"
VERSION_ID="16.04"
HOME_URL="http://www.ubuntu.com/"
SUPPORT_URL="http://help.ubuntu.com/"
BUG_REPORT_URL="http://bugs.launchpad.net/ubuntu/"
VERSION_CODENAME=xenial
UBUNTU_CODENAME=xenial
```

Build:

https://pastebin.com/TA3fH1B8

**Comment by luruke** [comment]
Oh stupid me, it was missing `pkg-config`.

I was able to get some example running (though with a warning `GLFW error 0x10008: x11: RandR gamma ramp support seems broken`).

I noticed performance are pretty low, `cross.frag` runs at 38fps in a small window.
I see that Raspbian has some specific build flag, is that for performance / GPU optimisation?

The GPU here should be a `ARM Mali-400 MP2`

ping @patriciogonzalezvivo
